modules/wmco-prerequisites.adoc

```diff
@@ -28,8 +28,8 @@ a|* Windows Server 2022 Long-Term Servicing Channel (LTSC). OS Build link:https:
 |Windows Server 2022 Long-Term Servicing Channel (LTSC). OS Build link:https://support.microsoft.com/en-us/topic/april-25-2022-kb5012637-os-build-20348-681-preview-2233d69c-d4a5-4be9-8c24-04a450861a8d[20348.681] or later.
 
 |Bare metal or provider agnostic
-|Windows Server 2022 Long-Term Servicing Channel (LTSC). OS Build link:https://support.microsoft.com/en-us/topic/april-25-2022-kb5012637-os-build-20348-681-preview-2233d69c-d4a5-4be9-8c24-04a450861a8d[20348.681] or later.
-
+a|Windows Server 2022 Long-Term Servicing Channel (LTSC). OS Build link:https://support.microsoft.com/en-us/topic/april-25-2022-kb5012637-os-build-20348-681-preview-2233d69c-d4a5-4be9-8c24-04a450861a8d[20348.681] or later.
+* Windows Server 2019, version 1809 Long-Term Servicing Channel (LTSC)
```

Comment:
I like how we have things in the [4.10 docs](https://docs.openshift.com/container-platform/4.10/windows_containers/windows-containers-release-notes-5-x.html#wmco-prerequisites-supported-5.1.0_windows-containers-release-notes), dropping the verbose `Long-Term Servicing Channel (LTSC)` descriptor

Something like
- Windows Server 2022 (OS Build [20348.681](https://support.microsoft.com/en-us/topic/april-25-2022-kb5012637-os-build-20348-681-preview-2233d69c-d4a5-4be9-8c24-04a450861a8d) or later).
- Windows Server 2019 (version 1809)

Reply:
IMO we can drop the `Long-Term Servicing Channel (LTSC)` for all entries under all the platforms